modular_skyrat/modules/modular_modsuit/module_of_holding.dm

```diff
@@ -0,0 +1,51 @@
+/obj/item/mod/module/storage/bluespace/boh/inert //Inert state of the BOH mod. Requires a refined bluespace core.
+	name = "Inert MOD wormhole storage module"
+	desc = "An inert MOD Wormhole Storage Module. Without its anomaly core, it is simply just a strangely ornate metal lattice with a circular socket in the bottom of it. With its core however, it becomes a way to store large quantities of items within a compacted bluespace wormhole, akin to a Bag of Holding."
+	icon_state = "module"
```

Comment:
no icon makes modules sad :(

Reply:
To be honest the generic module sprite looks pretty radical. I'm no spriter and I wont have someone sprite something unless im 110% sure it'll be added.